README.md

```diff
@@ -390,6 +390,9 @@ guaranteed that the `DELETE` will be processed first (and vice versa).
 Deleting a grouping key without metrics is a no-op and will not result
 in an error.
 
+The body of a POST or PUT request may be gzip-compressed. Add a header Content-Encoding: gzip to do so.
```

Comment:
How about adding another header here? E.g. `### Request compression`. Otherwise, this appears to be part of the documentation for the `DELETE` method.

Please enclose `Content-Encoding: gzip` in back ticks.